### DISCUSSION TITLE
Need to identify "correct" carriage returns

### DISCUSSION THREAD

**Discussion by enrac5** [jsvine/pdfplumber#1130] [Ask for help with specific PDFs]
[simple_table.pdf](https://github.com/jsvine/pdfplumber/files/15098858/simple_table.pdf)

I have a table that has some text in one of the columns (see attached). The text is split up into paragraphs (usually representing dialogue). I need to be able to correctly identify which sections of text are paragraphs. Right now, using the `extract_tables(...)` method, the text in the third column has line breaks for each line, which makes sense, but makes paragraph detection difficult. Any ideas on how I can correctly identify the separate blocks of text?

**Comment by enrac5** [comment]
Hi @jsvine any thoughts on this one?

**Comment by jsvine** [comment]
Passing `{ "text_layout": True }` may help. E.g.:

```python
table = page.extract_table({ "text_layout": True })
print(table[0][2])
```

Returns:

```
 CAMERA FOLLOWS IN
 FRONT OF MS LANE AS
 SHE WALKS.       
                  
 BEATLES (VO)     
 (overlaps) (singing) (through
 speakers) ...and shout!
                  
 MS LANE          
 (singing) ...twist and shout!
 (off) oy vay I’m tired. <sighs>
 (on) <hums> <grunts>
```

For more on how to adjust the layout parameters, see the `When layout=True` portion of the readme; keeping in mind that text-extraction parameters need to be prefixed with `text_` in the table extraction settings.
